Classes/GlobalStateExplorers/DatabaseBrowser/FLEXSQLiteDatabaseManager.m

```diff
@@ -59,6 +59,9 @@ - (BOOL)open {
         return [self storeErrorForLastTask:@"Open"];
     }
     
+    NSString *sql = @"VACUUM;"; // To flesh WAL file into sqlite
+    [self executeStatement:sql];
```

Comment:
Can you check the `PRAGMA schema.journal_mode` first and only do this when we are in the WAL mode?

Reply:
Hello `matrush`, how can I do this in Objective C ?

Reply:
It is basically the same as what you are doing already, just use `[self executeStatement:sql]` to execute the `PRAGMA schema.journal_mode`, and then check the result.

Checkout [this](https://github.com/FLEXTool/FLEX/blob/master/Classes/GlobalStateExplorers/DatabaseBrowser/FLEXSQLiteDatabaseManager.m#L108) as an example.

Reply:
@matrush I can add this, what do I use in place of `schema`?

Reply:
Yeah I think that might just work, maybe try and see what it returns?